mahotas/tests/LineDetectorByHough.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2008-2014, Jasson <1917098992@qq.com>
+# License: MIT (see COPYING file)
+
+import math
+import numpy as np
+
+class Line():
+    def __init__(self,r,angle):
+        self.r = r
+        self.angle = angle
+    def __eq__(self, other):
+        if self.r == other.r and self.angle == other.angle:
+            return True
+        else:
+            return False
+    def __hash__(self):
+        result = 31 + int(self.r)
+        result = 31 * result + int(self.angle * 100)
+        #print result
+        return result
+
+class Counter(dict):
```

Comment:
Isn't this just `collections.Counter`?

Reply:
Yes, I read the API, and try to using collections.Counter, it do better!